content/en/workflows/trigger.md

```diff
@@ -75,6 +84,10 @@ Each time the Notification Rule fires, it triggers a workflow run.
 
 To schedule a workflow run:
 1. On the workflow canvas, click **Add an Automated Trigger** and select **Schedule**.
+1. Add a service account:
+   1. Under **Configure**, add a service account next to **@worklow-** to dynamically create a service account and associate it with the workflow.
+   1. Select roles for your service account.
+   1. (Optional) If required, select permissions for the role.
```

Comment:
I think this may not need to be here.

That said, it is a bit involved: you'd have to go to all the connections used by actions in the workflow and ensure that the selected roles have resolve permissions.